Перенесен на сборщик проект из 1 модуля

AnnaSaushkina opened the pull request:
- Отделены элементы страниц (кроме кнопок),
- Не вставлены шрифты,
- Изменение содержимого работает, компоненты с разным содержимым вставляются.

AnnaSaushkina committed:
Перенесен на сборщик проект из 1 модуля

- Отделены элементы страниц (кроме кнопок),
- Не вставлены шрифты,
- Изменение содержимого работает, компоненты с разным содержимым вставляются.

Yan1993-Dev commented on source/blog.html:
```diff
@@ -0,0 +1,82 @@
+  <body>
+    @@include('blocks/header.html')
+    <main>
+        <section class="breadcrumbs container">
+            <h2 class="breadcrumbs__header">Blog</h2>
+            <ul class="breadcrumbs__list">
+                <li class="breadcrumbs__list-item arrow"><a href="/index.html">Home</a></li>
+                <li class="breadcrumbs__list-item active-page"><a href="/pages/blog.html">Blog</a></li> 
+            </ul>
+        </section>
```
Можно вынести в отдельный инклюд

Yan1993-Dev commented on source/blog.html:
```diff
@@ -0,0 +1,82 @@
+        <section class="blog container">
+            <h2 class="visually-hidden">blog</h2>
+            <ul class="blog__list">
+                <li class="blog__list-post">
+                    <img class="blog__list-post-img" src="/img/blog-1-chaotic-lego.jpg" alt="Chaotic lego" width="380" height="220">
+                    <p class="blog__list-post-date">23rd August 2018</p>
+                    <a href="#" class="blog__list-post-link">Top 5 fastest growing it service companies in philly</a>
+                    <p class="blog__list-post-text">Direct Line Development’s Team is excited to announce that we received UpCity’s 2019 Local Excellence Award! This prestigious award is annually presented to the Top 1% of service providers in the given marketplace.</p>
+                </li>
```
Тоже можно вынести в отдельный инклюд

Yan1993-Dev commented on source/catalog.html:
```diff
@@ -0,0 +1,87 @@
+                    "alt": "accessories",
+                    "modify": "accessories",
+                  })
+                <li class="categoryProducts__list-item sale">
+                    <img src="../img/sale.jpg" alt="sale" width="280" height="270">
+                    <a class="categoryProducts__list-item-text" href="#">
+                    SALE</a>
+                </li>
```
Доделай остальные элементы так же инклюдами

AnnaSaushkina committed:
Исправлены названия разметки и стилей баннера по БЭМ

AnnaSaushkina committed:
Добавлено отключение скролла при открытом модальном окне

AnnaSaushkina committed:
Merge branch 'main' of https://github.com/AnnaSaushkina/yo-saushkina-anna-2

Yan1993-Dev commented on source/views/blocks/footer.html:
```diff
@@ -1,5 +1,81 @@
 <footer class="footer">
-  <div class="container">
-    © 2022 Swimming Pool Plastering-Tile-Coping
+  <div class="style container">
```
класс style - не понятно о чем он говорит. Именуй если это блок, тем, чем он является

Yan1993-Dev commented on source/views/blocks/footer.html:
```diff
@@ -1,5 +1,81 @@
-  <div class="container">
-    © 2022 Swimming Pool Plastering-Tile-Coping
+  <div class="style container">
+      <section class="footer__menuColumn">
```
Мы обговаривали на занятии как именовать классы. 
footer__menuColumn -> footer__menu-сolumn

Yan1993-Dev commented on source/views/blocks/popup.html:
```diff
@@ -0,0 +1,31 @@
+
```
Нужно убрать лишние пустые строки

Yan1993-Dev commented on source/views/blocks/product.html:
```diff
@@ -0,0 +1,6 @@
+
```
Пустая строка

Yan1993-Dev commented on source/scss/style.scss:
```diff
@@ -15,46 +15,402 @@
+        line-height: 12px;
+        padding-left: 2px;
+    }
+    &__switch {
+        background-color: #D6CAC2;
+        width: 32px;
+        height: 3px;
+        border: none;
+    }
+ &__switch:hover {
+    cursor: pointer;
+    background-color: #725F5B;
+}
```
&__switch {
        background-color: #D6CAC2;
        width: 32px;
        height: 3px;
        border: none;
        &:hover {
          cursor: pointer;
          background-color: #725F5B;
}
}

Yan1993-Dev replied:
Можно сократить эту запись и оставить в таком варианте

AnnaSaushkina committed:
Исправлены названия по БЭМ, изменен синтаксис CSS на Sass, дополнены инклюды

- Оформлены классы основных блоков по БЭМ  (кроме страниц toys, popup),
- Исправлен синтаксис стилей на Sass (кроме страниц toys, popup). Не до конца оптимизированы повторяющиеся свойства и комбинированные селекторы.
- Добавлены инклюды: pagination, breadcrumbs. blog-post, prod, footer-item, toy-item, star, companies.
-Добавлено отключение скролла при открытии поп апа. Некорректно работает его включение при нажатии на область окна.
